### ISSUE TITLE
what is the difference between this and the community react-native-geolocation package currently? 

### ISSUE THREAD

**Issue by SymntxHomendra51** [Agontuk/react-native-geolocation-service#383]

**Comment by danidaryaweesh** [comment]
@SymntxHomendra51  I have tried this package and I can say that this package works more consistently across Android devices (both packages work well on iOS).

**Comment by SymntxHomendra51** [comment]
@danidaryaweesh okay i see the community version is updated recently and I think it is being developed very fast than this one. There watchposition method also has a timeout option, which this package lacks.

**Comment by danidaryaweesh** [comment]
@SymntxHomendra51 Yes that is true, however I'm encountering more issues with the community version (related to cached current location). So I decided to use this library for now.

**Comment by SymntxHomendra51** [comment]
ok i'm using this as well
